docs/source/api/tutorials/In Depth 3 - Drawing DiHypergraphs.ipynb

```diff
@@ -13,7 +13,7 @@
    "id": "ea8aad6a",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    xgi.draw_bipartite(DH)
</pre><p>The default coloring of hyperedges is a single color, instead of the crest cmap mapping edge size that we have in other functions. Any reason for this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/492/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Fixed!